packages/nextjs-template/components/DendronLayout.tsx

```diff
@@ -94,33 +99,26 @@ export default function DendronLayout(
       <Header
         style={{
           position: "fixed",
+          isolation: "isolate",
```

Comment:
huh, didn't realize this was a thing. so this is to make sure the header will be on top?

Reply:
Yes, this creates a new stack, the idea is not to worry about a potential z-index war. Each element in isolation with `z-index: 1` will always be in front of the rest of the content.